src/core_plugins/kibana/public/home/components/tutorial/introduction.js

```diff
@@ -0,0 +1,47 @@
+import './introduction.less';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Content } from './content';
+
+import {
+  EuiImage,
+} from '@elastic/eui';
+
+export function Introduction({ description, previewUrl, title }) {
+  let img;
+  if (previewUrl) {
+    img = (
+      <EuiImage
+        size="l"
+        hasShadow
+        allowFullScreen
+        title="screenshot of primary dashboard."
+        url={previewUrl}
```

Comment:
FYI you can use `fullScreenIconColor="dark"` to specify a dark icon which might stand out better against some of these images.

Reply:
And `title` should be `alt` ;)

Reply:
@cchaos Maybe the EuiImage examples need to be updated? This is the example I copied from Eui. Should title be set to alt here as well?
```
<EuiImage
    size="l"
    hasShadow
    allowFullScreen
    caption="Click me to see full screen"
    title="Accessible image title goes here"
    url="https://i.imgur.com/4qx7HhE.jpg"
  />
);
```

Reply:
@cchaos Never mind - looks like I had an old version of EuiImage component

Reply:
👍